cfn-resources/database-user/README.md

```diff
@@ -56,4 +56,5 @@ source <(${repo_root}/quickstart-mongodb-atlas/scripts/export-mongocli-config.py
 ${repo_root}/quickstart-mongodb-atlas/scripts/launch-x-quickstart.sh ${repo_root}/cfn-resources/database-user/test/databaseuser.sample-template.yaml SampleAccessList1 ParameterKey=ProjectId,ParameterValue=<YOUR_PROJECT_ID> ParameterKey=Username,ParameterValue=<USERNAME> ParameterKey=Password,ParameterValue=<PWD> 
 ```
 
-
+For more information please refer API Documentation,
+https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Database-Users
```

Comment:
Copy same format as footer in MongoDB::Atlas::Project reference to Documentation

Reply:
Handled in [6d075bc20598253c789ce5f833954a06a5fe750a](https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/132/commits/6d075bc20598253c789ce5f833954a06a5fe750a)